src/verification-needed/VerificationNeeded.tsx

```diff
@@ -0,0 +1,58 @@
+import { GlobalStyles } from "../styles";
+import styled from "styled-components";
+import SiteHeader from "../site-header/SiteHeader";
+
+const VerificationNeeded: React.FC = () => {
+  const refreshAndNavigateAway = () => {
+    window.location.href = "/";
+  };
+
+  const StyledButton = styled.button`
+    background: #00615c;
+    font-size: 16px;
+    border-radius: 12px;
+    color: white;
+    font-family: "Poppins", sans-serif;
+    height: 48px;
+    width: 200px;
+    outline: none;
+  `;
+
+  const ButtonWrapper = styled.div`
+    display: flex;
+    justify-content: center;
+  `;
+
+  return (
+    <div>
+      <div className="font-body text-green min-h-screen tracking-normal w-full">
+        <div className="max-w-screen-xl px-4 mx-auto">
+          <SiteHeader />
+          <main className="py-8 sm:py-16 sm:py-24">
+            <div className="max-w-3xl mx-auto px-6 sm:px-8 lg:px-10 flex flex-col justify-start">
+              <h1 className="text-2xl sm:text-3xl font-display text-left">
+                Almost There
+              </h1>
+              <p className="leading-7 text-base sm:text-lg my-6">
+                Look for a verification email in your inbox and click the link in that email.
+                After you verify your email, click the button below and you will be able to reach
+                the home page.
+              </p>
+              <ButtonWrapper className="leading-7 text-base sm:text-lg my-6">
+                <StyledButton label="Get started" onClick={() => refreshAndNavigateAway()}>Get started</StyledButton>
```

Comment:
I had to create this button that users on the verification page click after having verified their email because, for reasons I don't yet understand, simply clicking on the links on the page to go elsewhere does nothing after you've been forcibly redirected here (but they are fine if you navigate here manually). I think it has something to do with the way the routes and redirects are defined in `App.tsx` so if anyone has any ideas then let me know.

Reply:
assuming we wind up still needing this, instead of the click handler can this be a `<Link to="/" />`, as seen in the site header?

(I'm not really familiar with the router package we are using here so someone else may have a more satisfactory solution to the larger issue)

Reply:
I tried that to no avail, sadly.

Reply:
alas